### PR TITLE
Update ListItemTag component

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 4372 .

#### Changes proposed in this Pull Request:

- Update Showcase demo 
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by daileytj** [comment]
I can't get this running from a fresh clone of react-native-components with this as the showcase. I also noticed in the component library that the ListItemTag wasn't being exported in the top-level index.ts file in the md3 folder. @ektaghag-eaton Could you provide instructions on how you were able to run this along with a screenshot?

**Comment by ektaghag-eaton** [comment]
> I can't get this running from a fresh clone of react-native-components with this as the showcase. I also noticed in the component library that the ListItemTag wasn't being exported in the top-level index.ts file in the md3 folder. @ektaghag-eaton Could you provide instructions on how you were able to run this along with a screenshot?

I am getting below error
Screenshot 2023-09-07 at 4 40 42 PM

I thought it once the RN library PR merged it would work.

**Comment by ektaghag-eaton** [comment]
> I can't get this running from a fresh clone of react-native-components with this as the showcase. I also noticed in the component library that the ListItemTag wasn't being exported in the top-level index.ts file in the md3 folder. @ektaghag-eaton Could you provide instructions on how you were able to run this along with a screenshot?

raised PR https://github.com/etn-ccis/blui-react-native-component-library/pull/387. @daileytj , could you please review it?

**Review by daileytj — COMMENTED** [review]
Code looks fine here, but the base ListItemTag component may need work as these should not fill the entire width of the card it is in.
Simulator Screenshot - iPhone SE (3rd generation) - 2023-09-07 at 10 17 28

**Review by daileytj — APPROVED** [review]

File: components/kitchen-sink.tsx [hunk 1]
```diff
@@ -223,6 +223,13 @@ export const KitchenSink: React.FC = (): JSX.Element => {
                     />
                 </Card.Content>
             </Card>
+            <Card style={styles.card}>
+                <Card.Title title="ChannelValue" />
```
Comment (daileytj) [inline components/kitchen-sink.tsx]:
```suggestion
                <Card.Title title="ListItemTag" />
```

**Pull request merged**